__tests__/app.test.js

```diff
@@ -298,3 +298,110 @@ describe("POST /api/reviews/:review_id/comments", () => {
       });
   });
 });
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+//insert PATCH HERE
+
+describe("GET /api/users", () => {
+  test("GET: 200 - responds with a 200 status and an array of objects containing correct properties", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body }) => {
+        const { users } = body;
+        expect(users).toHaveLength(4);
+        users.forEach((user) => {
+          expect(user).toEqual({
+            username: expect.any(String),
+            name: expect.any(String),
+            avatar_url: expect.any(String),
+          });
+        });
+      });
+  });
+  test("GET: 404 - responds with a 404 error when .get is pathed incorrectly", () => {
```

Comment:
If you've already tested for this before, no need to do it for each endpoint. A non-existent endpoint is its own test, not specific to this one